capa/main.py

```diff
@@ -576,14 +543,6 @@ def handle_common_args(args):
     # disable vivisect-related logging, it's verbose and not relevant for capa users
     set_vivisect_log_level(logging.CRITICAL)
 
-    # py2 doesn't know about cp65001, which is a variant of utf-8 on windows
```

Comment:
Note that this code is not equivalent to the previous one in some versions of Python 3. Since Python 3.8 `cp65001` is an alias to `utf_8`. But not before Python 3.8 and it used to cause some problems: bugs.python.org/issue36778
Not sure if we should keep this code to ensure the same behavior in Python < 3.8. :thinking:

Reply:
yeah, if its an issue in 3.6, lets keep the code around and update the commen.t